modules/logging-loki-storage.adoc

```diff
@@ -0,0 +1,274 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+:_content-type: PROCEDURE
+[id="logging-loki-storage_{context}"]
+= Loki log storage
+
+The Loki Operator supports link:https://aws.amazon.com/[AWS S3], link:https://azure.microsoft.com[Azure], link:https://cloud.google.com/[GCS], link:https://min.io/[Minio],  link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[OpenShift Data Foundation], link:https://us.alibabacloud.com/[Alibaba Cloud] and link:https://docs.openstack.org/swift/latest/[Swift] for LokiStack object storage.
+
+The recommended nomenclature for Loki storage is `logging-loki-<type>`. General secret creation format is as follows:
+//Generic secret creation
+include::snippets/logging-create-secret-snip.adoc[]
+
+You can verify a secret has been created by running the following command:
+
+[source,terminal]
+----
+oc get secrets
```

Comment:
Does this need to mention our namespace (`-n openshift-logging`) or can we assume that the user knows where they created the secret?